mirror.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+
+# Dependencies: jq, curl, bash, rsync
+# Config settings
+OUTDIR='./repo'
+TMPDIR='./repo-tmp'
+# End of config settings
+
+rm -rf "${TMPDIR}"
+mkdir -p "${OUTDIR}" "${TMPDIR}"
+
+# TODO: Update to use the upstream repository
+RELEASE_INFO="$(curl -s https://api.github.com/repos/Doridian/archzfs/releases/latest)"
+
+FILE_INFO="$(echo "${RELEASE_INFO}" | jq '.assets | map(.browser_download_url + "|" + .updated_at) | join("\n")' -r)"
```

Comment:
I wonder if reading this in an array wouldn't be more foolproof, e.g.
```sh
readarray -t FILE_INFO < <(echo "${RELEASE_INFO}" | jq '.assets | map(.browser_download_url + "|" + .updated_at) | join("\n")' -r)
```
and iterate below with
```sh
for info in "${FILE_INFO[@]}"; do
```

Reply:
I haven't really used bash arrays at all, but yeah, that seems like a nicer solution, despite being a little more complex maybe? Yeah not sure, bash is never pretty to me

Reply:
Main advantage that comes to mind is that it would handle spaces in the entries gracefully. Not that we expect such spaces in our case, but maybe better safe than sorry. :D